src/rust/src/series/general.rs

```diff
@@ -132,4 +132,17 @@ impl PlRSeries {
     fn n_chunks(&self) -> Result<Sexp> {
         (self.series.n_chunks() as i32).try_into()
     }
+
+    fn chunk_lengths(&self) -> Result<Sexp> {
+        let out = self.series.chunk_lengths().collect::<Vec<usize>>();
+        let mut sexp = OwnedIntegerSexp::new(out.len())?;
+        for (i, out) in out.into_iter().enumerate() {
+            let _ = sexp.set_elt(i, out as i32);
+        }
+        Ok(sexp.into())
```

Comment:
Not sure about this, do we have a better way to convert a `Vec<usize>` to `Sexp`?

Reply:
Since usize may exceed the range of i32, I think we need to implement TryFrom for conversion to an Integer vector.

Reply:
We could replace `out as i32` by `i32::try_from(out)?`. Would that work for you?

Reply:
Good point, something like 48540ea seems to be sufficient.